<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CardEdit</class>
 <widget class="QWidget" name="CardEdit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>739</width>
    <height>454</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" rowstretch="2,5,2" columnstretch="1,1">
   <item row="0" column="1">
    <layout class="QGridLayout" name="menuLo">
     <item row="1" column="0">
      <widget class="QPushButton" name="newCtBtn">
       <property name="text">
        <string>New Cardtype</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QPushButton" name="newClBtn">
       <property name="text">
        <string>New Cardlist</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" rowspan="2">
      <widget class="QPushButton" name="loadFileBtn">
       <property name="text">
        <string>Load File</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2" rowspan="2">
      <widget class="QPushButton" name="saveFileBtn">
       <property name="text">
        <string>Save File</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="1">
    <widget class="QScrollArea" name="labelFieldsArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="labelFields">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>355</width>
        <height>234</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="labelLo">
       <property name="bottomMargin">
        <number>20</number>
       </property>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="2" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="newCardBtn">
       <property name="text">
        <string>New Card</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="saveValueBtn">
       <property name="text">
        <string>Save Value</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="removeCardBtn">
       <property name="text">
        <string>Remove Card</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0" rowspan="3">
    <layout class="QVBoxLayout" name="listLo" stretch="0">
     <property name="spacing">
      <number>10</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QListWidget" name="itemList">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
